cmd/kubeadm/app/constants/constants.go

```diff
@@ -461,6 +461,11 @@ func GetDNSIP(svcSubnet string) (net.IP, error) {
 	return dnsIP, nil
 }
 
+// GetLocalDNSIP returns a link-local DNS IP for nodelocal dns cache to listen for requests on
+func GetLocalDNSIP(svcSubnet string) (net.IP, error) {
```

Comment:
@kubernetes/sig-cluster-lifecycle-pr-reviews 
hi, @prameshj is this PR a WIP?
why declaration of GetLocalDNSIP if there are no call sites yet?

Reply:
This PR is ready for review, it adds support to enable this feature in e2e cluster scripts.
I just added this function to be used when this feature is supported in kubeadm.

Reply:
ok, since 169.254.25.10 and the feature as a whole, does not make sense yet for the cmd/kubeadm codebase possibly a TODO is needed in the comment above the function.

also please change `dns cache` -> `DNS cache`.

/assign @timothysc

Reply:
I will submit the kubeadm support changes in a follow up PR. I removed the unused function from the changes in this PR. I will fix the comment as you suggested.

Reply:
thank you!